src/appshell/qml/AboutDialog.qml

```diff
@@ -112,8 +112,12 @@ QmlDialog {
 
                 StyledTextLabel {
                     Layout.fillWidth: true
-                    text: qsTrc("appshell", "Copyright © 1999-2021 MuseScore BVBA and others.\nPublished under the GNU General Public License version 3.")
-                    enabled: false
+                    text: {
+                        var message = qsTrc("appshell", "Copyright © 1999-2021 MuseScore BVBA and others.<br>Published under the %1GNU General Public License version 3%2.")
```

Comment:
See above comment reg. HTML code in translatable text

Reply:
For some reason we need to keep the $copy, and can't replace it with its HLML equivalent `&copy;`? But do have to replace `\n` with `<br>`, pretty strange.

Reply:
I just found the solution for the `<br>` issue. It is actually identical to what 3.x used, just in qml rather than c++.